modules/ROOT/pages/classify-api-task.adoc

```diff
@@ -45,6 +45,11 @@ In environments, APIs can have more than a single instance. Therefore, an instan
 +
 image::classify-instance.png[]
 
+=== API Analytics Impact
+
+In the Pre-Crowd version of API Manager, all dashboards were organization-wide. In Crowd, each dashboard links to a particular environment. +
+If you had custom reports from those APIs before migrating, those custom reports would no longer work after you classify all your APIs.
```

Comment:
Unclear what "those" is referring to. Any API managed via API Manager that existed before Crowd?